docs/README.md

```diff
@@ -1,41 +1,38 @@
-# Website
+# ADIDAS MVI LIBRARY
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+## Introduction
+ADIDAS MVI is an efficient library that helps you implement the MVI architectural pattern in your Android applications. MVI is a pattern that promotes a unidirectional flow of data and provides a clear separation between the different components of your application. This library aims to simplify the implementation of MVI by providing a set of tools and utilities.
```

Comment:
The library isn't using the android SDK. 
It can be used in pure Kotlin and Java projects.